run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
```

Comment:
this should be `-lt 1`, so that large commands such as `./run.sh spark-submit blah blah blah` work too.